cardinal/gamestate/redis.go

```diff
@@ -12,10 +13,121 @@ import (
 	"github.com/rotisserie/eris"
 )
 
+type RedisStorage struct {
+	cachedClient  redis.Cmdable
+	currentClient redis.Cmdable
+}
```

Comment:
Add `var _ Storage = &RedisStorage{}` here to be explicit about what this struct is implementing.